src/ViewHelpers/DebugViewHelper.php

```diff
@@ -69,12 +89,12 @@ public static function renderStatic(array $arguments, \Closure $renderChildrenCl
         $typeOnly = $arguments['typeOnly'];
         $expressionToExamine = $renderChildrenClosure();
         if ($typeOnly === true) {
-            return (is_object($expressionToExamine) ? get_class($expressionToExamine) : gettype($expressionToExamine));
+            return (
+            \is_object($expressionToExamine) ? \get_class($expressionToExamine) : \gettype($expressionToExamine)
```

Comment:
Please avoid slash-prefixing function calls. Though you and I and a few others understand why this micro-optimisation would be done, explaining it to people who ask "what's this" becomes a pretty tedious task for a benefit that is nearly non-existing when the code actually runs.

This is also the reason why such a conversion was not applied globally to this repository.